source/identical.js

```diff
@@ -27,5 +27,23 @@ import _curry2 from './internal/_curry2.js';
  *      R.identical(0, -0); //=> false
  *      R.identical(NaN, NaN); //=> true
  */
-var identical = _curry2(_objectIs);
+var identical = function(a, b) {
+  switch (arguments.length) {
+    case 0:
+      return identical; 
+    case 1:
+      return (function(){
+        return function unaryIdentical(_b) {
+          switch (arguments.length) {
+            case 0:
+              return unaryIdentical;
+            default:
+              return _objectIs(a, _b);
+          }
+        };
+      }());
```

Comment:
Do we really need the IIFE and do we need the `case 0` case? I know that our `curry` implementation allows expressions such as `add()()(1)()()(2)` but I don't find that particularly helpful.

Reply:
> ... `add()()(1)()()(2)` but I don't find that particularly helpful.

Me too. I have *no idea* why an (n > 0)-ary Ramda function `f` must satisfy `f() ≡ f`. In years using Ramda have not used this deliberately. Perhaps @CrossEye can enlighten us.

We could easily get rid of the IIFE here, but I did not want to break things.